### ISSUE TITLE
Integrate Signup and Login APIs

### ISSUE THREAD

**Issue by AdnanMuzavor** [vigneshshettyin/EatMyURL#23]
### Description

Create a function for login and signup which will call the backend APIs.
Create Temporary interface for login and signup for calling these functions(To be designed later and no copy this time unlike before!!)
Show working of APIs using network section of chrome browser.

### Domain

Client - (React)

### Code of Conduct

- [X] I follow [Code of Conduct](https://github.com/vigneshshettyin/EatMyURL/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by AdnanMuzavor** [comment]
Shall I start working @vigneshshettyin ?

**Comment by vigneshshettyin** [comment]
Before that complete login, signup, & dashboard pages in react v2 note use material UI & styled components. You can create a new issue for this later once this is merged you can take up the integration issue. @AdnanMuzavor
